include/abb_librws/rws_poco_result.h

```diff
@@ -17,12 +16,12 @@ namespace abb :: rws
      * \brief A constructor.
      * 
      * \param http_status HTTP response status
+     * \param reason HTTP reason phrase
      * \param header_info HTTP response header info
      * \param content HTTP response content
      */
-    POCOResult(Poco::Net::HTTPResponse::HTTPStatus http_status,
+    POCOResult(Poco::Net::HTTPResponse::HTTPStatus http_status, std::string const& reason,
       Poco::Net::NameValueCollection const& header_info, std::string const& content);
```

Comment:
Can't we just pass `Poco::Net::HTTPResponse` as parameter and then extract from it status, reason and headers in initialization/getters?

Reply:
I also thought about it. This however introduces a dependency of `POCOResult` on `Poco::Net::HTTPResponse`.